app/src/main/java/com/github/wnder/Score.java

```diff
@@ -18,16 +19,17 @@
      * @param distanceReference radius, "permitted error" in KM
      * @return score
      */
-    public static double calculationScore(double distance, double distanceReference){
+    public static double calculationScore(double distance, double distanceReference, int radiusCircle){
 
         if(distance > distanceReference){
             return 0; //The given distance is bigger than what we expected with the reference
         }
 
         double normalized = distance / distanceReference; //a value between 1 and 0
         double sqrtDistribution = 1 - Math.sqrt(normalized);
+        double difficultyBonus = Math.pow(radiusCircle, 0.25);
 
-        return MAX_SCORE * sqrtDistribution;
+        return Math.floor(MAX_SCORE * sqrtDistribution * difficultyBonus);
```

Comment:
I think it would be better if the score stayed between 0 and 200 + if 200 points were still achievable when the radius was very low.
I was thinking about something in the lines of :

normalized = distance / (distanceReference * (maxRadius - radius) * arbitrary_factor);
sqrtDistribution = 1 - Math.sqrt(normalized);
return MAX_SCORE * sqrtDistribution;

Just my opinion though, and maybe what I say is not very clear/my math is wrong.

Reply:
I thought selecting a bigger radius means more points because if we apply your idea, when selecting 5km in radius the image is still really easy to guess which means most of the time, the harsher penalty would not be applied because the user still got max points.

Reply:
I was thinking of something like this, that would let user get maximum score with any radius, but with more difficulty:
distribution = 1-normalized^((radius^0.2)*0.2)
The ^0.2 and *0.2 can be changed, I just tried to have something that would seem reasonable with 5km to have an high score and still have "normalized^x" with x<1 for a 1000km radius

![image](https://user-images.githubusercontent.com/61224490/118470567-16329580-b707-11eb-93ed-c70014198c81.png)

Otherwise if we don't span value between 0 and MAX_SCORE I'd rather use @JeremyDezalos solution

Reply:
As you wish then